Makefile.am

```diff
@@ -92,6 +90,7 @@ rpm: local
 
 release: tag
 	$(MAKE) archive
+	$(MAKE) -C po/ $(PACKAGE_NAME).pot-update
```

Comment:
That should work correctly now, since po/Makefile now has all of the automatic dependencies from automake. So when you run make bumpver (without the autogen.sh step) followed by make archive, the first thing that make archive does is re-run auto* and ./configure to update the Makefiles, and .pot-update will then have the new $(PACKAGE_VERSION).

I'll update the PR to get rid of all-am and remove those two lines from bumpver.